projects/media-viewer/src/lib/toolbar/main-toolbar/search-bar/search-bar.component.ts

```diff
@@ -21,11 +19,16 @@ export class SearchBarComponent {
   haveResults = false;
 
   constructor(
-    public readonly toolbarButtons: ToolbarButtonVisibilityService
+    public readonly toolbarButtons: ToolbarButtonVisibilityService,
+    public readonly toolbarEvents: ToolbarEventService
   ) {}
 
+  public ngOnInit(): void {
+    this.toolbarEvents.searchResultsCount.subscribe(results => this.setSearchResultsCount(results));
```

Comment:
as before, don’t forget in these instances to unsubscribe in the ngOnDestroy

Reply:
added